src/components/ec-blog-item/ec-blog-item.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div>
+    <p>{{ title }}</p>
+    <p>{{ shortDescription }}</p>
+    <p>{{ author }}</p>
+    <p>{{ category }}</p> hello
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'EcBlogItem',
+  props: {
+    title: { type: String },
+    shortDescription: { type: String },
+    author: { type: String },
+    category: { type: String },
+  },
+};
+</script>
+<style>
+p {
+  font-size: 2em;
```

Comment:
We have predefined typography sizes in EBO design, we should be re-using them: https://chameleon.ebury.now.sh/?path=/docs/css-typography--basic.

What is the target font-size? e.g. if it's 16px and 24px line-height, we should use `body-text` typography. You can apply that using Tailwind by doing:
```
  @apply tw-body-text;
```

The full list of possible font-size/line-height combinations can be found here:
https://github.com/Ebury/chameleon/blob/master/tailwind.config.js#L610